https.go

```diff
@@ -177,7 +178,18 @@ func (proxy *ProxyHttpServer) handleHttps(w http.ResponseWriter, r *http.Request
 
 	case ConnectHijack:
```

Comment:
what's the difference between `ConnectAccept` and `ConnectHijack`? it doesn't seem like `ConnectHijack` actually makes a request to the backend so I'm not sure if `ConnectRespHandler` can do anything interesting. or are there fields on the `*ProxyCtx` set elsewhere?

Reply:
No, you're totally right and it belongs in AcceptConnect! I got myself confused because `Hijack()` is also the method you call to override the HTTP protocol on the http client and I didn't realize it was overloaded here